client/setup.py

```diff
@@ -41,7 +41,7 @@
     long_description_content_type='text/markdown',
     zip_safe=False,
     setup_requires=['setuptools>=18.0', 'wheel'],
-    install_requires=['jina>=3.7.0', 'docarray[common]>=0.13.0', 'packaging'],
+    install_requires=['jina>=3.7.0', 'docarray[common]>=0.13.14', 'packaging'],
```

Comment:
`0.13.14`? why?

Reply:
https://github.com/jina-ai/jina/blob/142c0bb1d9c2f3deaf04c60e3f6d3509390851f7/extra-requirements.txt#L35

Reply:
this can actually be handled by jina right? we can remove docarray requirement as long as we specify some recent version of jina

Reply:
Let's keep the bump on docarray.